changelog/13055.feature.rst

```diff
@@ -0,0 +1 @@
+``@pytest.mark.parametrize()`` and ``pytest.Metafunc.parametrize()`` now support the ``id_names`` argument enabling auto-generated test IDs consisting of parameter name=value pairs.
```

Comment:
At first glance, it seems that there are two different functions supporting `ids_name`, and perhaps only `@pytest.mark.parameterize()` should be show. Additionally, it would be very enticing if you added an example in the changelog, such as: #12492.

Reply:
Since both the mark decorator and the method on Metafunc now support `id_names`, I'd like to keep it that way if the maintainers don't have anything against this.

Regarding an example in the changelog: I've added an example to the documentation. Shouldn't that be enough? At least to me the changelog entry pretty much covers the new feature.